roxie/ccd/ccdprotocol.cpp

```diff
@@ -286,50 +331,14 @@ class ProtocolSocketListener : public ProtocolListener
         while (running)
         {
             Owned<ISocket> client = socket->accept(true);
-            Owned<ISecureSocket> ssock;
             if (client)
             {
-                if (streq(protocol.str(), "ssl"))
+                if (isSSL)
```

Comment:
I think we can remove this if (isSSL) block

Reply:
Just checking you were awake... or proving that I was not, or something...

Pushed another fix